pyat/at/collective/__init__.py

```diff
@@ -0,0 +1,11 @@
+# noinspection PyProtectedMember
+from ..lattice.options import _Dst
+
+# when atimplib was compiled with mpicc
+# this is required
+
+if(_Dst.mpi):
```

Comment:
It should be
```python
from ..lattice import DConstant
...
if DConstant.mpi:
```
`DConstant` is the public single instance of the private `_Dst` class. `_Dst` should not be refered to (except for building it).
Values in `DConstant` like steps, which are used as default values in functions, can be modified by the user or reset to their default values. But not `openmp` or `mpi`, which are read-only.

Reply:
Done